src/js/otus/components/OTUList.tsx

```diff
@@ -27,6 +27,9 @@ type OTUListProps = {
  */
 export default function OTUList({ match }: OTUListProps) {
     const { refId } = match.params;
+    const history = useHistory();
+    const location = useLocation<{ createOTU: boolean }>();
```

Comment:
Why handle this logic here instead of in CreateOtu? Both should have access to the same history object